lib/services/project-data-service.ts

```diff
@@ -124,5 +124,13 @@ export class ProjectDataService implements IProjectDataService {
 			projectFilePath
 		};
 	}
+
+	@exported("projectDataService")
+	public getNsConfigDefaultContent(data: Object): string {
+		const config: INsConfig = {};
+		Object.assign(config, data);
+
+		return JSON.stringify(config);
```

Comment:
Can we use `return JSON.stringify(data);` If it is possible `config` will not be needed.

Reply:
The main idea of the method is to merge default values with some overrides. For now nsconfig's default state is an empty object but writing the method this way makes it a bit more readable.